subsys/emds/Kconfig

```diff
@@ -16,6 +16,13 @@ menuconfig EMDS
 
 if EMDS
 
+config NEW_EMDS
+	bool "Enable new EMDS implementation"
+	help
+	  Enable the new implementation of Emergency Data Storage.
+	  This is temporary option for debug purpose.
+	  It will be removed in the future.
```

Comment:
removed as in there will only be the new one and the new will be removed from the filenames?